src/cache/lru-cache.spec.ts

```diff
@@ -61,4 +61,19 @@ describe('LRUCache', () => {
     expect(oneCache.get('a')).toBeFalsy();
     expect(oneCache.get('b')).toBe('banana');
   });
+
+  /**
+   This test case might be an overkill but in case Map() changes,
```

Comment:
probably handled by a test like your "should evict oldest entry when capacity limit is reached" but no harm no foul